services/metadata/src/metadata/routes/domain/template.clj

```diff
@@ -1,10 +1,12 @@
 (ns metadata.routes.domain.template
-  (:use [compojure.api.sweet :only [describe]])
+  (:use [compojure.api.sweet :only [describe]]
+        [korma.core])
   (:require [schema.core :as s])
   (:import [java.util Date UUID]))
 
 (def TemplateIdPathParam (describe UUID "The metadata template ID"))
 (def AttrIdPathParam (describe UUID "The metadata attribute ID"))
+(def ValidValueTypeEnum (apply s/enum (map #(:name %) (select :value_types (fields :name)))))
```

Comment:
I think it would be a good idea to extract this (map ... (select ...)) code into the the `metadata.persistence.templates` namespace, to keep all of our database access logic together.